File: D.S.A/hashing/Greater-than-one.js
function CountDigits(arr){

    let map =new Map();

    for( let num of arr){
        if(map.has(num)){
            map.set(num,map.get(num)+1)
        }else{
            map.set(num,1);
        }
    }

    return map;

}

let initialResult= CountDigits([1,1,1,2,2,2,2,23,3,3]);

let result=[];
for(let [key,value] of initialResult){
    if(value>1){
        result.push(key);
    }
}

console.log(result);

File: Java/dsa/arrays/easy/hashing/GreaterThanOne.java
import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

public class GreaterThanOne {
    public static List<Integer> duplicates(int[] arr) {
        Map<Integer, Integer> map = new HashMap<>();
        for (int num : arr) {
            map.put(num, map.getOrDefault(num, 0) + 1);
        }
        List<Integer> result = new ArrayList<>();
        for (Map.Entry<Integer, Integer> entry : map.entrySet()) {
            if (entry.getValue() > 1) {
                result.add(entry.getKey());
            }
        }
        return result;
    }
}
